Dockerfile

```
@@ -7,13 +7,13 @@
 # where <VERSION> is the Docker image version to create.
 
 # generic Fuseki image
-FROM openjdk:8u312-jdk-bullseye AS fuseki
+FROM eclipse-temurin:8u462-b08-jdk-jammy AS fuseki
```

```suggestion
FROM eclipse-temurin:8-jdk-jammy AS fuseki
```

@peichman-umd, I would like to use the label with only the major version so that every build will get the latest patches.